src/range-to-string.js

```diff
@@ -0,0 +1,68 @@
+/* global Node */
+
+/**
+ * Return the next node after `node` in a tree order traversal of the document.
+ */
+function nextNode(node, skipChildren) {
+  if (!skipChildren && node.firstChild) {
+    return node.firstChild
+  } else if (node.nextSibling) {
+    return node.nextSibling
+  }
+  while (node.parentNode) {
+    node = node.parentNode
+    if (node.nextSibling) {
+      return node.nextSibling
+    }
+  }
+  /* istanbul ignore next */
```

Comment:
Alternatively, I think it could be written like this:

```js
if (!skipChildren && node.firstChild) {
  return node.firstChild
}

do {
  if (node.nextSibling) {
    return node.nextSibling
  }
  node = node.parentNode
} while(node)

return node
```